Cargo.toml

```diff
@@ -293,153 +293,153 @@ testnet-runtime = [
-sp-session = { git = "https://github.com/NunoAlexandre//substrate", branch = "polkadot-v0.9.43" }
-sp-staking = { git = "https://github.com/NunoAlexandre//substrate", branch = "polkadot-v0.9.43" }
-sp-state-machine = { git = "https://github.com/NunoAlexandre//substrate", branch = "polkadot-v0.9.43" }
-sp-std = { git = "https://github.com/NunoAlexandre//substrate", branch = "polkadot-v0.9.43" }
-sp-storage = { git = "https://github.com/NunoAlexandre//substrate", branch = "polkadot-v0.9.43" }
-sp-timestamp = { git = "https://github.com/NunoAlexandre//substrate", branch = "polkadot-v0.9.43" }
-sp-tracing = { git = "https://github.com/NunoAlexandre//substrate", branch = "polkadot-v0.9.43" }
-sp-transaction-pool = { git = "https://github.com/NunoAlexandre//substrate", branch = "polkadot-v0.9.43" }
-sp-trie = { git = "https://github.com/NunoAlexandre//substrate", branch = "polkadot-v0.9.43" }
-sp-version = { git = "https://github.com/NunoAlexandre//substrate", branch = "polkadot-v0.9.43" }
-sp-wasm-interface = { git = "https://github.com/NunoAlexandre//substrate", branch = "polkadot-v0.9.43" }
-substrate-build-script-utils = { git = "https://github.com/NunoAlexandre//substrate", branch = "polkadot-v0.9.43" }
-substrate-frame-rpc-system = { git = "https://github.com/NunoAlexandre//substrate", branch = "polkadot-v0.9.43" }
-substrate-prometheus-endpoint = { git = "https://github.com/NunoAlexandre//substrate", branch = "polkadot-v0.9.43" }
-substrate-wasm-builder = { git = "https://github.com/NunoAlexandre//substrate", branch = "polkadot-v0.9.43" }
-substrate-state-trie-migration-rpc = { git = "https://github.com/NunoAlexandre//substrate", branch = "polkadot-v0.9.43" }
-substrate-test-client = { git = "https://github.com/NunoAlexandre//substrate", branch = "polkadot-v0.9.43" }
-try-runtime-cli = { git = "https://github.com/NunoAlexandre//substrate", branch = "polkadot-v0.9.43" }
-substrate-test-runtime = { git = "https://github.com/NunoAlexandre//substrate", branch = "polkadot-v0.9.43" }
-substrate-test-runtime-client = { git = "https://github.com/NunoAlexandre//substrate", branch = "polkadot-v0.9.43" }
-node-primitives = { git = "https://github.com/NunoAlexandre//substrate", branch = "polkadot-v0.9.43" }
-pallet-nomination-pools = { git = "https://github.com/NunoAlexandre//substrate", branch = "polkadot-v0.9.43" }
-sc-service-test = { git = "https://github.com/NunoAlexandre//substrate", branch = "polkadot-v0.9.43" }
-sc-network-transactions = { git = "https://github.com/NunoAlexandre//substrate", branch = "polkadot-v0.9.43" }
-sp-consensus-beefy = { git = "https://github.com/NunoAlexandre//substrate", branch = "polkadot-v0.9.43" }
-sp-consensus-grandpa = { git = "https://github.com/NunoAlexandre//substrate", branch = "polkadot-v0.9.43" }
+binary-merkle-tree = { git = "https://github.com/paritytech//substrate", branch = "polkadot-v0.9.43" }
```

Comment:
@NunoAlexandre not sure why you used your fork here, however, this is one of the main reasons why I addressed these changes in this separate PR.

Reply:
We can't compile the official version, failing with the error: 

```
warning: `pallet-block-rewards` (lib) generated 2 warnings (run `cargo fix --lib -p pallet-block-rewards` to apply 1 suggestion)
     Compiling liquidity-pools-gateway-routers v0.0.1 (/Users/n/centrifuge/centrifuge-chain/pallets/liquidity-pools-gateway/routers)
  error[E0599]: no function or associated item named `assert_last_event` found for struct `frame_system::Pallet` in the current scope
    --> /Users/n/.cargo/git/checkouts/substrate-f4423eebfa0046a3/5e49f6e/frame/system/benchmarking/src/lib.rs:55:16
     |
  55 |         System::<T>::assert_last_event(frame_system::Event::<T>::CodeUpdated.into());
     |                      ^^^^^^^^^^^^^^^^^ function or associated item not found in `Pallet<T>`
```


I tried using subalfred and featalign to fix any potential mixing feature flags and tried using different rust toolchains but couldn't get any further. So my fork basically just silences that line on `frame-system-benchmarking` so that we can continue with the work on our side. We can always get back to it at the end.